src/main/java/org/threadly/litesockets/Client.java

```diff
@@ -484,6 +503,249 @@ protected void addRead(final int size) {
      * @param client This is the client the close is being called for.
      */
     public void onClose(Client client);
-  }  
+  }
+  
+  /**
+   * ClientOptions that can be changed depending on what kind of client you want.
+   * 
+   * In general these should not be set unless you have a very specific use case.
+   * 
+   * @author lwahlmeier
+   *
```

Comment:
you could consider adding an `@since 3.1.0` if you want others to know when an API became available